resources/js/features/game-list/components/SimilarGameSuggestionsMainRoot/SimilarGameSuggestionsMainRoot.tsx

```diff
@@ -0,0 +1,50 @@
+import { router } from '@inertiajs/react';
+import { type FC, memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuDices, LuUser } from 'react-icons/lu';
+
+import { BaseButton, baseButtonVariants } from '@/common/components/+vendor/BaseButton';
+import { GameHeading } from '@/common/components/GameHeading';
+import { InertiaLink } from '@/common/components/InertiaLink';
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+import { GameSuggestionsDataTable } from '../GameSuggestionsDataTable';
+
+export const SimilarGameSuggestionsMainRoot: FC = memo(() => {
+  const { sourceGame } = usePageProps<App.Platform.Data.GameSuggestPageProps>();
+
+  const { t } = useTranslation();
+
+  const game = sourceGame as App.Platform.Data.Game;
+
+  const handleReload = () => {
+    router.reload({ only: ['paginatedGameListEntries'] });
+  };
+
+  return (
+    <div>
+      <GameHeading game={game}>{t('Game Suggestions')}</GameHeading>
+
+      <div className="flex flex-col gap-2">
+        <div className="flex justify-end rounded bg-embed p-2">
+          <div className="flex gap-2">
+            <InertiaLink
+              href={route('game.suggestions.personalized')}
+              className={baseButtonVariants({ size: 'sm', className: 'gap-1' })}
+            >
+              <LuUser className="size-4" />
+              {t('See games picked just for you')}
```

Comment:
I think this needs some additional text. Since we're on a page for finding "games similar to X", this suggests that it would further filter it down to "games just for you similar to X".

I'm not coming up with any good suggestions on how to improve it though.

Reply:
Shooting from the hip, here's four ideas:
* "Explore games based on your history"
* "View your personal game matches"
* "View all your recommended games"
* "See your full recommendation feed"

Do you like any of these?

Reply:
The last two are good.

I like the wording on the third, but the last seems to be the most accurate description.

Maybe "View personal suggestions from our entire library"?

Reply:
I like it, but it may be too long:
![Screenshot 2025-01-16 at 8 17 04 PM](https://github.com/user-attachments/assets/bc1ab582-4a9e-4c4d-9e8f-c68bcd482b91)

I went with option 4, happy to change it though:
![Screenshot 2025-01-16 at 8 17 53 PM](https://github.com/user-attachments/assets/af215de9-7b30-4b0e-92cb-1ca30fd4874b)